llvm/utils/repo/Makefile

```diff
@@ -0,0 +1,121 @@
+#
+# Makefile for building LLVM libraries using musl-prepo libc (requires GNU make)

```

Comment:
> 
> 
> Is there a compelling reason why we have this makefile here rather than in a separate repository (as the standard C library is)?
> 

The Makefile is used to build LLVM runtime libraries. All runtime libraries source code is in llvm-project-prepo repository. Therefore I put the Makefile here. Any suggestion? 

> Talking about “building LLVM libraries” is confusing because all of LLVM is a collection of libraries! Perhaps using the phrase “LLVM _runtime_ libraries” more accurately describes them.

Good point, changed in commit: https://github.com/SNSystems/llvm-project-prepo/pull/145/commits/7ef52d812cc1e524350d3241e5bdf90077da4b3e

Reply:
> Any suggested repository?

Personally, I would create a single repository which contains the materials and instructions for building all of the runtime libraries.